site/app/controllers/PollController.php

```diff
@@ -482,7 +484,8 @@ public function submitEdits(): RedirectResponse {
         }
 
         $em->flush();
-
+        $web_socket_message = ['type' => 'poll_updated', 'poll_id' => $poll_id, 'socket' => 'student', 'message' => 'Poll updated'];
```

Comment:
It's generally more readable to wrap large associative arrays.  An additional benefit of doing so is that the diff will be cleaner if someone edits the array in the future--the same reason why a trailing comma is useful.  The same applies below as well.

```suggestion
        $web_socket_message = [
            'type' => 'poll_updated',
            'poll_id' => $poll_id,
            'socket' => 'student',
            'message' => 'Poll updated',
        ];
```